Kde-neon.md

```diff
@@ -17,16 +25,18 @@ More information about downloading. Whether there are different versions availab
 
 ## Reviews
 
-#### Review title
+#### Differences from Kubuntu
 
 ```
-Review message here
+Because Kubuntu has the KDE Plasma suite on a Ubuntu base OS, KDE neon often is mistaken for Kubuntu and vice versa. However, the primary difference between the two operating systems is that Kubuntu maintains stable releases and LTS version of Ubuntu while KDE neon focuses on updating developer editions of KDE applications without maintaining stable releases of Ubuntu unless the root user actively chooses to upgrade their systems.
 ```
-- Reviewer Name, Date
+- Muhammed Hashim, 29-10-2018
 
-#### Review 2 title
+#### Graphical user interface
 
 ```
-Review 2's message here
+The graphical user interface (GUI),The interface which replace cryptic command by their graphical representation is called GUI, it is a form of user interface that allows users to interact with electronic devices through graphical icons and visual indicators such as secondary notation, instead of text-based user interfaces, typed command labels or text navigation. GUIs were introduced in reaction to the perceived steep learning curve of command-line interfaces (CLIs), which require commands to be typed on a computer keyboard.
```

Comment:
This is not a review and more of a general description of GUI. Please remove this review